docs/dapp/opl/frontend.md

```diff
@@ -12,9 +12,8 @@ simply apply a sparse checkout of the complete frontend repo. Inside your
 
 ```shell
 git init .
-git remote add -f origin https://github.com/oasisprotocol/playground
```

Comment:
I didn't want to mess with the `origin` branch so I rather used `demo-opl-secret-ballot` one.

Reply:
No problem. I should just give the tutorial another pass.